api/jwt/token_blacklist_test.go

```diff
@@ -0,0 +1,123 @@
+package jwt
```

Comment:
Yay tests! I still don't get it, that github.com/portainer/portainer does not provide unit tests for the API. Also: these tests read excellent! :+1: